charts/quantum-serverless/charts/gateway/values.yaml

```diff
@@ -46,11 +41,41 @@ application:
     channel: "ibm_quantum"
     url: "https://auth.quantum-computing.ibm.com/api"
 
-database:
-  host: postgresql
-  port: 5432
-  user: bn_keycloak
-  name: bitnami_keycloak
+cos:
+  claimName: gateway-claim
+
+secrets:
+  secretKey:
+    create: true
+    name: gateway-secret-key
+    key: django-secret-key
+    value: "django-insecure-&)i3b5aue*#-i6k9i-03qm(d!0h&662lbhj12on_*gimn3x8p7"
+  servicePsql:
+    create: true
+    name: service-psql-binding
+    key:
+      databaseName: database-name
+      databasePassword: database-password
+      host: database-host
+      port: database-port
+      userName: user-name
+    value:
+      databaseName: gateway_db
+      databasePassword: CHANGE_ME
+      host: postgresql
+      port: 5432
+      userName: bn_keycloak
+  superuser:
+    create: user
```

Comment:
This is overridden so it may not matter but it should be true or false.

Reply:
Ups, yep!

Reply:
Solved in [a2c898f](https://github.com/Qiskit-Extensions/quantum-serverless/pull/928/commits/a2c898fb2188c2bc4d916ea478cfb86d95cd3cf1)